sound/soc/sof/pcm.c

```diff
@@ -301,8 +301,14 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 		ipc_first = true;
 		break;
 	case SNDRV_PCM_TRIGGER_PAUSE_RELEASE:
+		if (pcm_ops && pcm_ops->trigger_ipc_first &&
+		    substream->stream == SNDRV_PCM_STREAM_PLAYBACK)
+			ipc_first = true;
 		break;
 	case SNDRV_PCM_TRIGGER_START:
+		if (pcm_ops && pcm_ops->trigger_ipc_first &&
+		    substream->stream == SNDRV_PCM_STREAM_PLAYBACK)
```

Comment:
The commit message should probably be reworded. It starts with a generic description of a problem, without mentioning any specific direction, and ends with a statement that capture is just fine, which doesn't quite add up with the notion that the sequence is broken.